src/views/tenants.js

```diff
@@ -0,0 +1,138 @@
+import React, { Component } from 'react';
+import BootstrapTable from 'react-bootstrap-table-next';
+import paginationFactory from 'react-bootstrap-table2-paginator';
+import UserContext from '../UserContext';
+import { Link } from "react-router-dom"
+// import Accordion from '../components/Accordion';
+import * as axios from 'axios';
+
+const columns = [{
+    dataField: 'name',
+    text: 'Name',
+    sort: true,
+    headerStyle: () => {
+      return { width: "20%" };
+    }
+  }, {
+    dataField: 'property',
+    text: 'Property',
+    sort: true,
+    headerStyle: () => {
+      return { width: "20%" };
+    }
+  }, {
+    dataField: 'address',
+    text: 'Join Staff',
+    sort: true,
+    headerStyle: () => {
+      return { width: "20%" };
+    }
+  }, {
+    dataField: 'tenants',
+    text: 'Added On',
+    sort: true,
+    headerStyle: () => {
+      return { width: "20%" };
+    }
+}];
+
+const selectRow = {
+  mode: 'checkbox',
+  clickToSelect: true,
+  sort: true,
+  headerColumnStyle: () => {
+    return { width: "5%" };
+  }
+};
+
+const pageButtonRenderer = ({
+  page,
+  active,
+  disable,
+  title,
+  onPageChange
+}) => {
+  const handleClick = (e) => {
+    e.preventDefault();
+    onPageChange(page);
+  };
+  const activeStyle = {};
+  if (active) {
+    activeStyle.backgroundColor = 'black';
+    activeStyle.color = 'white';
+  } else {
+    activeStyle.backgroundColor = 'gray';
+    activeStyle.color = 'black';
+  }
+  if (typeof page === 'string') {
+    activeStyle.backgroundColor = 'white';
+    activeStyle.color = 'black';
+  }
+  return (
+    <li className="page-item">
+      <a href="#" onClick={ handleClick } style={ activeStyle }>{ page }</a>
+    </li>
+  );
+};
+
+const options = {
+  pageButtonRenderer
+};
+
+export class Tenants extends Component {
+    constructor(props) {
+        super(props);
+        
+        this.state = {
+        tenants: [],
+        }
+
+        // this.getTenants = this.getTenants.bind(this)
+    }    
+
+    componentDidMount() {
+        // this.getTenants(this.context);
+    }
+
+    // getTenants = (context) => {
+    //     axios.get(`${process.env.REACT_APP_API_URL}/tenants`, { headers: {"Authorization" : `Bearer ${context.user.accessJwt}`} })
```

Comment:
Note the axios requests will need to look like `axios.get("/api/tenants")` (see https://github.com/codeforpdx/dwellingly-app/pull/177)